core/node/fwd.hpp

```diff
@@ -14,7 +14,7 @@ namespace boost {
 }  // namespace boost
 
 namespace libp2p {
-  class Host;
+  struct Host;
```

Comment:
What benefits of it?

Reply:
`libp2p::Host` is actually a `struct`, compiler complained about that struct/class inconsistency